mkdocs.yml

```diff
@@ -114,6 +114,7 @@ nav:
     - Nodes: integrations/nodes/
     - Trigger nodes: integrations/trigger-nodes/
     - Credentials: integrations/credentials/
+    - Custom API operations: integrations/custom-operations.md
```

Comment:
This title is a touch broad, since it covers everything the HTTP node does (and anything in nodes you create).

What about 'Custom actions for existing nodes'?

Reply:
Plus we're following the language the UI uses, which is always a bonus